apps/web/modules/event-types/views/event-types-single-view.tsx

```diff
@@ -383,9 +383,14 @@ const EventTypePage = (props: EventTypeSetupProps) => {
     ),
   });
   const {
+    reset,
     formState: { isDirty: isFormDirty, dirtyFields },
   } = formMethods;
 
+  useEffect(() => {
```

Comment:
When the values are synced with global limits, and we come back to individual event screen, it doesn't get updated because we are setting the defaultValues initially and once the defaultValues gets updated, the form doesn't get updated so we reset the form with updated values